app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,7 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="white">#ffffff</color>
+    <color name="mainBackground">#8acee3</color>
+    <color name="color_transparent">#50FFFFFF</color>
```

Comment:
be consistent with the naming conventions, there is mixed camel and snake case here